### ISSUE TITLE
File paths using Sam Meehan's EOS area no longer exist

### ISSUE THREAD

**Issue by matthewfeickert** [hsf-training/hsf-training-cms-analysis-snapshot-stats#8]
Hi. :wave: I'm opening this Issue as this was raised first on the [ATLAS Discourse forums for ATLAS RECAST](https://atlas-talk.web.cern.ch/t/kerberos-in-gitlab-ci/644) (internal link, sorry). It was noticed that

https://github.com/hsf-training/hsf-training-cms-analysis-snapshot-stats/blob/a3b51d9bd9a9d3d372d8c28ed838017beac799e4/.gitlab-ci.yml#L11

uses a file path that is under Sam Meehan's EOS user area, but as Sam left the field a few years ago his CERN EOS user area no longer exists. If this file path could get updated to one that isn't controlled by a single user that would be great.

**Comment by klieret** [comment]
Good point (sorry that this hasn't been addressed yet), I'll bring it up on the next meeting regarding the analysis preservation workshop
